tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -60,3 +74,45 @@ def test_define_dataset(self):
         # Post conditions
         self.assertIsInstance(results, bigquery.Dataset)
         self.assertEqual(results.labels, self.label_or_tag)
+
+    def test_update_labels_and_tags(self):
+        # Tests if project_id is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.dataset_id,
+                          self.label_or_tag)
+
+        # Tests if dataset_id is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.project_id,
+                          self.label_or_tag)
+
+        # Tests if no label or tag is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.project_id,
+                          self.dataset_id)
+
+        # Pre-conditions
+        results = define_dataset(self.project_id, self.dataset_id,
+                                 self.description, self.label_or_tag)
+
+        self.client = bigquery.Client()
```

Comment:
Variables `self.dataset` and `self.client` are not used outside this method and should be declared locally (i.e. `dataset` and `client`).